sdk/turing/router/router.py

```diff
@@ -144,7 +144,7 @@ def update(self, config: RouterConfig) -> "Router":
         self._config = config
         updated_router = Router.from_open_api(
             turing.active_session.update_router(
-                router_id=self.id, router_config=config.to_open_api()
+                router_id=self.id, router_version_config=config.to_open_api().config
```

Comment:
This is a breaking change to the users. Could you document it in the PR description? Ideally we can keep the API backward-compatible but in this instance, I feel that it may be ok to force the users to update the SDK after the app is released, so let's remember to document it in the release notes later.